### PR TITLE
new lesson basic SELECT postgresql

### PR DESCRIPTION
**Pull request by linda-seiter**
aggregates and joins in separate lesson

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,31 +2,818 @@
...
+ORDER BY [comma-separated list of columns]
+LIMIT [count];
+```
 
-## Topic 2
+- The `WHERE` clause is optional.  The clause is used to select a subset of rows that match
+  the predicate.  If omitted, all rows are retrieved.
+- The `ORDER BY` clause is optional.  The clause is used to sort the result set based on
+  the order of one or more columns.  If omitted, the rows are not sorted and may appear in
+  any order.
```
Comment (kcleland0818) [inline README.md]: I thought it ordered by the primary key by default if `ORDER BY` was omitted? Or is that not always the case with Postgres?

File: README.md [hunk 2]
```diff
@@ -2,31 +2,818 @@
...
+- The `LIMIT` clause is optional. The clause restricts the number of rows displayed in the result.
+
+
+## Table Setup  (Code Along)
+
+We will work with the same `pet` table from the previous lesson.  
+Copy and execute the following SQL statements in the**pgAdmin**
```
Comment (kcleland0818) [inline README.md]: Missing space between "the" and "**pgAdmin"**

File: README.md [hunk 3]
```diff
@@ -2,31 +2,818 @@
...
+
+
+
+<tr>
+<td>
+Display name and age of
+pets between 4 and 10 years old (inclusive):
+
+```sql
+SELECT name, breed, age
```
Comment (kcleland0818) [inline README.md]: Remove breed from the query to match the instruction and output.

File: README.md [hunk 4]
```diff
@@ -2,31 +2,818 @@
...
+
+<tr>
+<td>
+The NULL keyword is used to
+test for missing or unknown values:
+
+```sql
+SELECT id, name, age
+FROM pet
+WHERE age IS NULL
```
Comment (kcleland0818) [inline README.md]: Missing semicolon

File: README.md [hunk 5]
```diff
@@ -2,31 +2,818 @@
...
+
+<tr>
+<td>
+Use NOT NULL to
+test for known values:
+
+```sql
+SELECT id, name, age
+FROM pet
+WHERE age IS NOT NULL
```
Comment (kcleland0818) [inline README.md]: Missing semicolon

File: README.md [hunk 6]
```diff
@@ -2,31 +2,818 @@
...
+</tr>
+
+</table>
+
+
+
+## ORDER BY
+
+The `ORDER BY` clause sorts the retrieved rows by order of one or more columns.
+The optional keywords `asc` and `desc` will sort respectively
```
Comment (kcleland0818) [inline README.md]: I usually uppercase ASC and DESC to show that it is a SQL operation. It looks like you capitalized them in the learning goals. For consistency, can we either change the casing here or in the learning goals please?

**Review by kcleland0818 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (linda-seiter) [reply README.md]: Does not seem to be case with Postgres.  It lists based order
of most inserted/update. Weird and unexpected!

File: README.md (hunk 3 above)
Comment (linda-seiter) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (linda-seiter) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (linda-seiter) [reply README.md]: Does not seem to be case with Postgres. It lists based order
of most recent insert/update. Weird and unexpected!

File: README.md (hunk 4 above)
Comment (linda-seiter) [reply README.md]: fixed

File: README.md (hunk 5 above)
Comment (linda-seiter) [reply README.md]: fixed

File: README.md (hunk 6 above)
Comment (linda-seiter) [reply README.md]: thanks for catching this.  queries fixed to capitalize.

**Review by kcleland0818 — APPROVED** [review]

**Review by kcleland0818 — APPROVED** [review]
Thanks for the added clarification!

**Review by developerjaym — APPROVED** [review]

**Review by kcleland0818 — APPROVED** [review]

**Review by AlveeM — APPROVED** [review]

**Pull request merged**